package.json

```diff
@@ -38,11 +39,16 @@
     "js-yaml": "^3.10.0",
     "nodemailer": "^4.0.1",
     "nodemailer-sendgrid-transport": "^0.2.0",
+    "rehype-stringify": "^3.0.0",
```

Comment:
that's a lot of stuff to just convert markdown to html (there are simpler packages like marked). 
that would be fine anyway if we'll need this tools for more complex processing in the near future.

Reply:
yes I selected rehype because it will make it easier to pre-process and post-process the content (needed for https://www.pivotaltracker.com/story/show/151434506 and https://www.pivotaltracker.com/story/show/151284725)